src/components/Sidebar/Sidebar.jsx

```diff
@@ -45,6 +58,14 @@ export default ({
             </div>
           );
         })}
+        <div className="sidebar__docs-version">
+          You are reading webpack {currentDocsVersion} documentation. Change here to:
+          <ul>
+            {docs.filter(item => item.version !== currentDocsVersion).map(item => <li key={`webpack-${item.version}-docs`}>
+              <a rel="nofollow" href={item.url}>webpack {item.version} documentation</a>
```

Comment:
Why `nofollow`?

Reply:
i am afraid of SEO sanctions for duplicated content, ideally we add robots.txt with disallow in v4 for all the pages except for homepage what you think?

Reply:
Let's see how it goes from Google and user search perspective